README.md

```diff
@@ -4,6 +4,8 @@
 
 18F is conducting research to identify opportunities for developing a web-based eligibility rules service that could be used to make eligibility determinations for a human services program. This _Phase 2_ work follows a prior _Phase 1_ investigation sprint that identified creating such a service as worthy of further exploration. This repository documents Phase 2.
 
+> Presentation: **[Exploring an Eligibility Rules Service](https://github.com/18F/eligibility-rules-service/files/1627427/eligibility-rules-service-explanation.pdf)**
```

Comment:
Yay! I wonder if it's worth posting a version with the notes included, [like this](https://drive.google.com/file/d/1AAfygpR5U_8klbXWidfqV-dgDzHG3fQh/view), since so much of the meat of the content is in the speaker notes?

Reply:
Yes, of course you're right. I posted the wrong version. Updating now.

Reply:
sweet! looks great!